MaybeGame/Player.cs

```diff
@@ -4,279 +4,86 @@ namespace MaybeGame
-        {
-            // Agility is the ability to move quickly and easily.
-            var agility = 5;
-
-            switch (race)
-            {
-                case "Human":
-                    agility += 3;
-                    break;
-
-                case "Dwarf":
-                    agility += 2;
-                    break;
-
-                case "Elf":
-                    agility += 7;
-                    break;
-
-                case "Undead":
-                    agility += 1;
+                    Race = PlayerRaces.Undead;
                     break;
 
                 default:
-                    Console.WriteLine("Error 404: Race not found");
+                    Console.WriteLine("This doesn't work!");
                     break;
             }
 
-            return agility;
-        }
-        private int SetDexterity(string race)
-        {
-            // Dexterity is the readiness and grace in physical activity.
-            var dexterity = 5;
-
-            switch (race)
-            {
-                case "Human":
-                    dexterity += 3;
-                    break;
-
-                case "Dwarf":
-                    dexterity += 2;
-                    break;
-
-                case "Elf":
-                    dexterity += 5;
-                    break;
-
-                case "Undead":
-                    dexterity += 2;
-                    break;
-
-                default:
-                    Console.WriteLine("Error 404: Race not found");
-                    break;
-            }
-
-            return dexterity;
-        }
-        private int SetHealth(int stamina)
-        {
-            return 50 + (stamina * 3);
-        }
-        private int SetDefence(int stamina)
-        {
-            return 20 + stamina;
-        }
-        private double SetDodgeChance(int dexterity)
-        {
-            var maxDodgeChance = 0.95d;
-
-            if (DodgeChance >= maxDodgeChance)
-            {
-                DodgeChance = maxDodgeChance;
-                Console.WriteLine("Dodge chance maxed out(stop adding points to Dexterity)!");
-            }
-            else
-            {
-                DodgeChance += dexterity * 0.005;
-            }
-
-            return DodgeChance;
-        }
-        private int SetSpeed(int agility)
-        {
-            return 20 + (agility * 2);
-        }
-        private int SetAttackPower(int strength)
-        {
-            return 15 + (5 * strength);
-        }
-        public void IsLevelUpPossible(double experiencePoints)
-        {
-            if (experiencePoints >= 1)
-            {
-                Level++;
-                Console.WriteLine("Level up! Congratulations\n" +
-                    $"Your level is now {Level}");
-                ExperiencePoints -= 1;
-            }
-            else
-            {
-                ExperiencePoints += experiencePoints;
-            }
+            return Race;
```

Comment:
Returning Property of a class? Methods should only return scope specific variables